siac/walletcmd.go

```diff
@@ -75,11 +75,11 @@ By default the wallet encryption / unlock password is the same as the generated
 	}
 
 	walletLoadSiagCmd = &cobra.Command{
-		Use:   `siag [filepaths]`,
-		Short: "Load a siag keyset into the wallet",
-		Long: `Load a set of siag keys into the wallet - typically used for siafunds.
-Example: 'siac wallet load siag key1.siakey,key2.siakey'`,
-		Run: wrap(walletloadsiagcmd),
+		Use:     `siag [filepath,...]`,
```

Comment:
I don't think this is the right way to specify that multiple filepaths can be used. Qubes does it this way: https://www.qubes-os.org/doc/vm-tools/qvm-copy-to-vm/ (note that their whole style is different from ours, therefore not the best example)

Another alternative seems to be `siag [filepath]...`